companies/models.py

```
@@ -119,6 +119,46 @@ class FinancialStatement(models.Model):
+        null=True,
+        blank=True,
+        verbose_name="EPS (주당순이익)",
+        help_text="당기순이익 ÷ 발행주식수",
+    )
+    operating_profit_margin = models.DecimalField(
+        max_digits=10,
+        decimal_places=2,
+        null=True,
+        blank=True,
+        verbose_name="영업이익률",
+        help_text="(영업이익 ÷ 매출액) × 100",
```

Ruff (0.14.13)

String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)